test/Azure.Functions.Cli.Tests/Azure.Functions.Cli.Tests.csproj

```diff
@@ -36,11 +36,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Folder Include="Properties\" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <None Include="..\..\.nuget\NuGet.Config" Link="NuGet.Config">
+    <None Include="..\..\NuGet.Config" Link="NuGet.Config">
```

Comment:
So this was an accidental change I made when resolving merge conflicts during the cherry-pick, but I'm wondering if it's actually a "good" accident. Both NuGet.Config files exist, but the `..\..\NuGet.Config` file is newer and also follows the latest standards for nuget. Perhaps someone forgot to port this change into v3 earlier? Let me know if you're okay with this, or if you want me to revert it.

Reply:
Thanks for the catch, Chris. Yes, I think we should just use the `..\..\NuGet.Config` file. I'm fine with this change